Scripts/Linux/InstallFromSource.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# To test on a Debian-based Linux machine:
+#
+# ./InstallFromSource.sh
+# (outputs Git, GCM, and Scalar versions)
+#
+# Small clone should work:
+# scalar clone https://dev.azure.com/gvfs/ci/_git/ForTests
+#
+# Big clone for authenticated repo should work:
+# scalar clone <url>
+#
+# Note: if you are using a headless Linux machine, then GCM Core will
+# not be able to store your credentials. Instead, create a PAT and
+# use that for password (multiple times). This will be fixed soon.
+
+# Nuke previous data
+rm -rf git-vfs.deb gcm-core scalar log.txt
+
+echo "Installing dotnet SDK 3.1"
+yes | sudo apt-get install dotnet-sdk-3.1 dotnet-runtime-3.1
+
+echo "Installing Git from microsoft/git"
+curl -L https://github.com/microsoft/git/releases/download/v2.28.0.vfs.0.0/git-vfs_2.28.0.vfs.0.0.deb >git-vfs_2.28.0.vfs.0.0.deb
+sudo dpkg -i ./git-vfs_2.28.0.vfs.0.0.deb
+
+echo "Installing Watchman from apt-get"
+sudo apt-get install Watchman 2>1 >>log.txt
```

Comment:
```suggestion
sudo apt-get install Watchman 2>&1 >>log.txt
```

Reply:
Also, I'm not sure if your intent here is to put apt's stderr to the user's stdout, or to combine them both in the log. If the latter, then you need to swap the order of redirections.

Reply:
I'm removing those logs, anyway. But thanks!